common/cards/card-plugins/single-use/mending.js

```diff
@@ -0,0 +1,63 @@
+import singleUseCard from './_single-use-card'
+import {GameModel} from '../../../../server/models/game-model'
+import {validPick} from '../../../../server/utils/reqs'
+
+/**
+ * @typedef {import('common/types/pick-process').PickRequirmentT} PickRequirmentT
+ *
+ */
+
+// TODO - Must work with Gemintay ability to use two single use cards per turn (should mend the first one)
```

Comment:
With the new functionality, this should work fine, so this todo can be removed - provided you've set up canAttach correctly -see below